pages/linux/goaccess.md

```diff
@@ -0,0 +1,20 @@
+# goaccess
+
+> GoAccess is an open source real-time web log analyzer.
```

Comment:
As the command name is in the title, it doesn't need to be included in the description. 👍🏻 🙂 
```suggestion
> An open source real-time web log analyzer.
```